modules/nf-core/software/fastqc/functions.nf

```diff
@@ -0,0 +1,59 @@
+/*

```

Comment:
Isn't this generic and could thus be used centrally for more than just one module?

Reply:
Yup, it is the same but in order to keep module installs independent from each other it has to be shipped with the module file itself. e.g. say you want to update `fastqc` and not `star` but the `functions.nf` file for `fastqc` has changed on nf-core/modules then you can update `fastqc` and things won't break with `star` too and more importantly things are still reproducible because you are using the same bundled files for a particular git commit.